appyx-components/stable/backstack/common/src/commonMain/kotlin/com/bumble/appyx/components/stable/backstack/BackStack.kt

```diff
@@ -1,8 +1,8 @@
-package com.bumble.appyx.components.backstack
+package com.bumble.appyx.components.stable.backstack
```

Comment:
I wonder if we actually need stable in the package name? `backstack`, `testdrive`, etc are unique anyway

Reply:
+1, I'd rather not have `stable` in the package name. (Let's keep it for the experimental and internal ones though!)